mindmeld/models/taggers/pytorch_crf.py

```diff
@@ -516,15 +530,15 @@ def fit(self, X, y):
         del X, y, train_X, train_y, dev_X, dev_y, stratify_tuples
         gc.collect()
 
-        self.build_params(self.encoder.num_feats, self.encoder.num_classes)
+        self.build_params(*self._encoder.get_feats_and_classes())
 
         if self.optimizer == "sgd":
             self.optim = optim.SGD(self.parameters(), lr=0.01, momentum=0.9, nesterov=True, weight_decay=1e-5)
         if self.optimizer == "adam":
-            self.optim = optim.Adam(self.parameters(), weight_decay=1e-5)
+            self.optim = optim.Adam(self.parameters(), lr=0.001, weight_decay=1e-5)
```

Comment:
Do we really want these hard-coded?

Reply:
I don't anticipate us ever having to change those values in the future, and also I don't know if a user would ever care about those values enough for us to move them into constant variables. For readability's sake, I am inclined to keep them as is.